src/notifier/index.js

```diff
@@ -1,8 +1,14 @@
+require('dotenv').config();
+
 const nodemailer = require('nodemailer');
 const { getUsersToNotify } = require('../database');
 
-const { SENDGRID_USERNAME: user, SENDGRID_PASSWORD: pass } =
-  process.env.NODE_ENV === 'production' ? process.env : require('./config');
+let user = '';
+let pass = '';
+if (process.env.NODE_ENV === 'production') {
+  user = process.env.SENDGRID_USERNAME;
+  pass = process.env.SENDGRID_PASSWORD;
+}
```

Comment:
Do we need to check for `production`? Shouldn't these be read from `process.env` always?

Reply:
@adeolaawoyemi You are right we shouldn't need to check for prod. I just kept it without considering that. In fact I'll update so that is there is no username and password it will fallback to use the test account.